apps/condo/domains/common/utils/serverSchema/resolveHelpers.js

```diff
@@ -0,0 +1,45 @@
+const { get } = require('lodash')
+
+const { FLAT_UNIT_TYPE, SECTION_SECTION_TYPE } = require('@condo/domains/property/constants/common')
+
+
+const getUnitTypeFieldResolveInput = ({ unitTypeFieldName = 'unitType', unitNameFieldName = 'unitName' } = {}) => ({ resolvedData, existingItem }) => {
+    const newItem = { ...existingItem, ...resolvedData }
+    const unitType = get(newItem, unitTypeFieldName)
+    const unitName = get(newItem, unitNameFieldName)
+    const existedUnitType = get(existingItem, unitTypeFieldName)
+
+    if (!unitName && unitType) {
+        return null
+    } else if (!unitType && unitName) {
+        if (existedUnitType) {
+            return existedUnitType
+        }
+        return FLAT_UNIT_TYPE
```

Comment:
If we've decided to make 'flat' the default value, describe this decision within the field's schemaDoc.

Reply:
ok, I add it to docs, but this is not a new change, it has always been the default unless otherwise stated